robottelo/datafactory.py

```diff
@@ -461,3 +467,29 @@ def get_valid_preupgrade_data(test_entity, field):
         raise InvalidArgumentError('No such field/test \'{0}\' exists in YAML'
                                    'file under {1}'.format(field, test_entity))
     return datalist
+
+
+def gen_html_with_total_len(length=10):
```

Comment:
I guess @sghai mean moving to fauxfactory? I agree, fauxfactory can handle that generator.